packages/core/src/components/fields/SchemaField.js

```diff
@@ -16,7 +16,6 @@ import {
   deepEquals,
   getSchemaType,
 } from "../../utils";
-import UnsupportedField from "./UnsupportedField";
 
 const REQUIRED_FIELD_SYMBOL = "*";
 const COMPONENT_TYPES = {
```

Comment:
Do you mind adding the list of fields you can override (which are defined here) to the docs, and add UnsupportedField to it? https://react-jsonschema-form.readthedocs.io/en/latest/advanced-customization/#customizing-the-default-fields-and-widgets